knife/lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -272,6 +272,22 @@ def win_wget_ps
           escape_and_echo(win_wget_ps)
         end
 
+        def bootstrap_certstore_cert_ps
+          bootstrap_certstore_cert_ps = <<~EOH
+            $pfx_password = New-Object -TypeName PSObject
+            # cspell:disable-next-line
+            $pfx_password | Add-Member -MemberType ScriptProperty -Name "Password" -Value { ("~!@#$%^&*_-+=`|\\(){}[<]:;'>,.?/0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz".tochararray() | Sort-Object { Get-Random })[0..14] -join '' }
+            if (-not (Test-Path HKLM:\\SOFTWARE\\Progress)){
```

Comment:
This key could be there, but still without the Authenticator key present.

Reply:
Corrected!